docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -95,12 +95,13 @@ Settings related to typography.
 | customFontSize | boolean | true |  |
 | fontStyle | boolean | true |  |
 | fontWeight | boolean | true |  |
+| fluid | object |  |  |
 | letterSpacing | boolean | true |  |
 | lineHeight | boolean | false |  |
 | textDecoration | boolean | true |  |
 | textTransform | boolean | true |  |
 | dropCap | boolean | true |  |
-| fontSizes | array |  | name, size, slug |
+| fontSizes | array |  | fluidSize, name, size, slug |
```

Comment:
Wondering if we should make it `__experimentalFluidSize`?

Reply:
For consistency, I _think_ some of the existing Typography controls like font family / letter spacing, etc, use the `__experimental` prefix in the setting within `block.json` but in `theme.json` the experimental prefix doesn't appear to be used? I imagine that might be so that if the feature is stabilised, themes don't need to update their `theme.json` file?

Reply:
Updated to use `__experimental*` 🙇

Reply:
I believe @andrewserong is correct and was suggesting here that it should **not** be `__experimentalFluid` in the theme.json.

Reply:
Yes, I think it looks like it we only use an `__experimental` prefix in `block.json` (e.g. if we're looking up whether or not the block supports the feature) rather than `theme.json` (where we're grabbing values from)?

Reply:
Theme.json has a schema migration mechanism to it that gives us more freedom there should settings be added, removed, or renamed.

Reply:
Oh great, thanks. I got mixed up. Easy fix.